servatrice/servatrice.ini.example

```diff
@@ -107,6 +107,9 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Disallow usernames containing these words
+disallowedwords="admin,administrator"
```

Comment:
Wouldn't `disallowedwords="admin"` be identical to this line? (since "administrator" contains "admin" anyway)